src/components/RuleCard/index.jsx

```diff
@@ -662,8 +662,15 @@ function RuleCard({ rule, onRuleDelete, ...props }) {
             <div className={classes.scheduledChangesHeader}>
               <Typography
                 className={classes.scheduledChangesTitle}
+                component="h4"
```

Comment:
Out of curiosity, why use an `h4` here, but an `h2` for other Rules?

Reply:
In the UI the rule name has a bigger font and a bold weight. It is purely semantics.